corpus/extracted/0001-initial-extraction/README.md

```diff
@@ -0,0 +1,152 @@
+* message bafy2bzaceacixqbswmycl7z7ffwvo3w52p542sfo7jgalanpmhd6qntml3z6y: vector generation aborted (receipt mismatch)
+* message bafy2bzacebkiannwx26vmxkwwkc5uylaorenam5za4xuz4spslkhlvu6abi5s: vector generation aborted (receipt mismatch)
+* message bafy2bzacebo56aphmu4eb4fvljiy6p3fwu4rc6i2yixhylnciv7nm4xm27kjq: vector generation aborted (receipt mismatch)
+* message bafy2bzacecg64l52m3wq3pppe7q4rp5oaymofdkgom7vdrofxdplf3g5xeviw: vector generation aborted (receipt mismatch)
+* message bafy2bzaceaq6ysuhcpd4lhjp7cc6caqkhyx7hbnnoqh5il622bgiqtqyxo3s2: vector generation aborted (receipt mismatch)
+* message bafy2bzacebe2kzhdjjtquj5yof32o3oop2ffubhsoqchn42jlezhtvpisqnf6: vector generation aborted (receipt mismatch)
+* message bafy2bzaced7k7wygzqianzymmnyeb56flcen33mzurax5ok3jjeondwmg2o7g: vector generation aborted (receipt mismatch)
+* message bafy2bzacebj67kfde3yn2gcse6rwuz4ko67fj4ai4xciylcgwuqyd6wvhf6ak: vector generation aborted (receipt mismatch)
+* message bafy2bzacec7ceff2l7slqjdsptidd73nuy7kmiqecyqmufhgnalwy4dbvc5tm: vector generation aborted (receipt mismatch)
+* message bafy2bzaced2tjkxc77s34hajk4ckylh3zgpkoojypwexrr5eamdwi527lbnha: vector generation aborted (receipt mismatch)
+* message bafy2bzacedehjkw77xl7sc2fqh6o3v63phs4z2ijawtujyxxzq4sw6s6ait46: vector generation aborted (receipt mismatch)
+* message bafy2bzaceack4i2a73uav3juaxrae2yt3lcztjmd5tg7osttrfamawmfc6lyo: vector generation aborted (receipt mismatch)
+* message bafy2bzacebte3egdqgx6qnvsh7ubgnkjxu7hxzcdzfjhuioplb56eeb5zvzls: vector generation aborted (receipt mismatch)
+* message bafy2bzacecpb6h5ahvmt5etmcug2zmzkoozk74oypmbejxl4cgbqpf7syyies: vector generation aborted (receipt mismatch)
+```
+
+## SQL query (against sentinel/visor)
+
+Resulted in [selection.csv](./selection.csv).
+
+```sql
+with uniq_msgs as (
+    select msgs.cid                     as message_cid,
+           actors.code                  as receiver_code,
+           msgs.method                  as method_num,
+           receipts.exit_code           as exit_code,
+           b.height                     as height,
+           b.cid                        as block_cid,
+           row_number()
+           over (partition by msgs.cid) as uniq_rn
+    from public.messages as msgs -- join messages, with their blocks, their actor types, and receipts.
+             join public.block_messages as block_msgs on msgs.cid = block_msgs.message
+             join public.block_headers as b on b.cid = block_msgs.block
+             join public.actors as actors
+                  on msgs.to = actors.id and actors.state_root = b.parent_state_root -- this is not precise, but actor types are immutable, so it'll suffice
+             join public.receipts as receipts on msgs.cid = receipts.message
+    where b.height >= 65000
+     and b.height <= 69000 -- process only heights above epoch 50000; chainwatch only synced up to 60063.
```

Comment:
Comment seems wrong, considering the values in the query.

Reply:
Fixed, thanks for catching!